docs/UserGuide.md

```diff
@@ -175,6 +175,15 @@ Exits the program.
 
 Format: `exit`
 
+### Pagination
+
+The app comes with a pagination at the bottom of the list. It supports navigation to a specific page or navigation to 
+the next or previous page. Each page contains a maximum of **10** items. The pagination supports a maximum of **10** pages
+being displayed at one time, to access more pages, one can go to the last page and clicks on `NEXT` page button to access other pages.
```

Comment:
Since users are expected to be very good at keyboard typing (I guess so since Command Line Interface), maybe we can say that users can also press left and right keys if preferred (though they have to at least click on the pagination once before the keys will work).

Reply:
> Since users are expected to be very good at keyboard typing (I guess so since Command Line Interface), maybe we can say that users can also press left and right keys if preferred (though they have to at least click on the pagination once before the keys will work).

Sure, I will include this in the next commit.